tests/queries/0_stateless/01277_random_fixed_string.sql

```diff
@@ -0,0 +1 @@
+SELECT DISTINCT c > 30000 FROM (SELECT arrayJoin(arrayMap(x -> reinterpretAsUInt8(substring(randomFixedString(100), x + 1, 1)), range(100))) AS byte, count() AS c FROM numbers(100000) GROUP BY byte ORDER BY byte);
```

Comment:
And a test for return type (toTypeName)
and negative test when argument is zero
and negative test when argument is not constant.

Reply:
Done.